firewalld/ceph-installer.xml

```diff
@@ -2,5 +2,5 @@
 <service>
   <short>Ceph Installer HTTP service</short>
   <description>The service allows you to install and configure ceph via a RestFul API that accepts and returns JSON responses along with meaningful error codes and messages.</description>
-  <port protocol="tcp" port="8181"/>
+  <port protocol="tcp" port="8080"/>
```

Comment:
we should not default to port 8080 I think because that port is commonly used by other HTTP services. The collision is unlikely but far less chances for that if we keep 8181